src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -9,12 +9,14 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="~/TheAdmin/Styles/TheAdmin.css" />
+    <link rel="stylesheet" asp-src="~/OrchardCore.Resources/Styles/jquery-ui.min.css" />
```

Comment:
Is it really necessary here? Or just as one the pages that actually use this control?

Reply:
Good catch, it looks I forgot this one during test